GraphQL/InputMapper/SearchQueryMapper.php

```diff
@@ -14,65 +21,17 @@ public function mapInputToQuery(array $inputArray)
         $query = new Query();
         $criteria = [];
 
-        if (isset($inputArray['ContentTypeIdentifier'])) {
-            $criteria[] = new Query\Criterion\ContentTypeIdentifier($inputArray['ContentTypeIdentifier']);
-        }
-
-        if (isset($inputArray['Text'])) {
-            foreach ($inputArray['Text'] as $text) {
-                $criteria[] = new Query\Criterion\FullText($text);
+        foreach ($inputArray as $inputField => $inputValue) {
+            if ($criteriaMapper = $this->existsCriteriaMapperForField($inputField)) {
+                $criteria = array_merge($criteria, $this->criteriaMappers[$inputField]->map($inputValue));
```

Comment:
It feels a bit weird that you're using a method to check if there is a visitor for that field at line 25, and then use the index from the `criteriaMappers` array. You may as well use `isset` to check if there's a visitor.